content/markdown/concepts/emphasis/emphasis.md

```diff
@@ -1,5 +1,6 @@
 ---
 Title: 'Emphasis'
+Description: 'Emphasis can be added by making text bold or italic. To bold text, add two asterisks  or underscores __ before and after a word or phrase. To bold the middle of a word for emphasis, add two asterisks without spaces around the letters. markdown bold text*  markdown bold text*'
```

Comment:
This is actually an artifact of a bug lol

Can just remove everything after `letters.`